### PR TITLE
yast2_i: Prevent wasting time with check_screens

### PR DESCRIPTION
**Pull request by okurz**
Related needles PR:
https://github.com/os-autoinst/os-autoinst-needles-opensuse/pull/121

Verification run: http://lord.arch/tests/5333

### CODE REVIEW

**Comment by okurz** [comment]
test fails because I deleted a file which has maintainer information missing. We could change the test but I guess it is actually easier to add maintainer before removing :-P

File: tests/console/yast2_i.pm [hunk 1]
```diff
@@ -72,20 +72,22 @@ sub run() {
...
-        send_key 'alt-f';
-    }
+    push @tags, 'yast2-sw_shows_summary' unless get_var('YAST_SW_NO_SUMMARY');
+    do {
+        assert_screen(\@tags);
+        # automatic changes for manual selections
+        if (match_has_tag('yast2-sw-packages-autoselected')) {
+            wait_screen_change { send_key 'alt-o' };
+        }
+        elsif (match_has_tag('yast2-sw_automatic-changes')) {
```
Comment (dzedro) [inline tests/console/yast2_i.pm]: this could be in one condition "or" ?

Comment (okurz) [inline tests/console/yast2_i.pm]: yes, thank you. updated

**Comment by okurz** [comment]
I guess there is a problem with the size of this PR? Do I need to split it?

**Comment by sysrich** [comment]
you should be in training - and we shouldn't be merging PR's from you while you're in training ;)

**Comment by okurz** [comment]
On Wednesday 26 October 2016 01:02:40 Richard Brown wrote:
> you should be in training - and we shouldn't be merging PR's from you while
> you're in training ;)

then there shouldn't be submissions to the acceptance group while we are in 
training ;-)

**Comment by Soulofdestiny** [comment]
> then there shouldn't be submissions to the acceptance group while we are in
training ;-)

so we should stop working when you are not there?

**Comment by okurz** [comment]
> so we should stop working when you are not there?

yes, that's the logical consequence :-)

**Comment by okurz** [comment]
done, only related changes

**Comment by okurz** [comment]
merge, anyone?

**Comment by coolo** [comment]
https://openqa.suse.de/tests/633831#step/yast2_i/15 - hurray for more needles!

**Comment by okurz** [comment]
but in that screen it is not finished, it didn't return properly. maybe just increase timeout

**Comment by coolo** [comment]
Well, the post_fail_hook can type commands into it, so I claim it *is* finished - just not showing the prompt.

**Comment by okurz** [comment]
And I claim https://openqa.suse.de/tests/633831#step/yast2_i/17 shows that the typing occurs before the prompt is reactive again -> increase timeout

**Pull request merged**